code/game/objects/items/weapons/clown_items.dm

```diff
@@ -204,13 +202,21 @@
 	icon_state = "glue0"
 
 	w_class = W_CLASS_TINY
-
 	var/spent = 0
+	var/uses = -1 //As opposed to reworking the spent var, just avoiding confusion. Set to -1 for superglue.
+	var/glue_duration = -1 //-1 For infinite.
+	var/list/allowed_glue_types = list(
+		/obj/item,
+		/obj/structure/bed,
+	)	
 
 /obj/item/weapon/glue/examine(mob/user)
 	..()
 	if(Adjacent(user))
-		user.show_message("<span class='info'>The label reads:</span><br><span class='notice'>1) Apply glue to the surface of an object<br>2) Apply object to human flesh</span>", MESSAGE_SEE)
+		if(glue_duration < 0) //We assume we're dealing with superglue, its not really badcode.
```

Comment:
why would you not just put this into the schoolglue's examine?

Reply:
I guess you're right, I just didn't want to clutter the chat with both messages showing up at once.